netmanager/.env.sample

```diff
@@ -13,4 +13,5 @@ REACT_APP_CLOUDINARY_PRESET=abcdef
 REACT_APP_MAPBOX_TOKEN=pk.ey123abc
 REACT_APP_JIRA_KEY=abcdef
 NODE_PATH=src/
-PORT=5000
\ No newline at end of file
+PORT=5000
+REACT_APP_GOOGLE_ANALYTICS_TOKEN='G-HJTULVGQ8W'

```

Comment:
Can we remove the enclosing quotes @Codebmk?

Reply:
Those have no effect honestly. Thanks.

Reply:
Is there a trimming mechanism to get rid of them in the codebase?

Reply:
Look at the .env.sample as a documentational file. It helps new users setup their env values basing on what they see. That's why its best to have the (dummy)value exactly as is supposed to be.